data/autoyast_sle15/containers/autoyast_x390s_sle15sp1.xml

```diff
@@ -0,0 +1,396 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns"
+	 xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url><![CDATA[https://updates.suse.com/SUSE/Products/SLE-Module-Desktop-Applications/15-SP1/s390x/product?WaKaU4v5azj4vUscFcSfSlGZwM3mBK-IPwFtBBeiQhoBCDepdbD4_s9T-TLCkRWxPVfftslVB89sh6OWyEDJqXqmJaNrbhqdJbaDODv_UhCGb4zlcmXe86pE0530Ru137Afw3xd7_Ozp0NAcUk1Qwr79hosw6__NojZKSe0H0v-L]]></media_url>
+        <product>sle-module-desktop-applications</product>
+        <product_dir/>
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[https://updates.suse.com/SUSE/Products/SLE-Module-Server-Applications/15-SP1/s390x/product?ECZH2kIsyLn3gMu2bKBYatqEbgG5wwFLievb0lvVDs--P3oJPh6A-4s_o4iQuN9Jp9a1CyAsiVU36og7Mz_0KzkKAOInJbtGpnU90l5O_iiGBHvljqzPoZaYU1CKdEefjxkOmpZodr0Rrrba3E3FxGs5euufGqw5Wc71Y01jCzo]]></media_url>
+        <product>sle-module-server-applications</product>
+        <product_dir/>
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[https://updates.suse.com/SUSE/Products/SLE-Module-Basesystem/15-SP1/s390x/product?UG4gh9ztGTN2IGEGCQfMCC42nyYkMrKjATBJRbw3Bh0tv2NKZ1hk0n2jvNx83sZf5qJoo2VoYD29inhbBpUhwQP4m4AdSH6-EWReRZ3h2H0X6XXJpLVY7Ak6bBJ2MRE_1YUnEsNKYIU1yKLtOvxe3hEXznTiG28]]></media_url>
+        <product>sle-module-basesystem</product>
+        <product_dir/>
+      </listentry>
+    </add_on_products>
+  </add-on>
+  <bootloader>
+    <global>
+      <append>TERM=linux console=ttyS0 console=ttyS1 resume=/dev/disk/by-path/ccw-0.0.0000-part3 crashkernel=163M mitigations=auto</append>
+      <cpu_mitigations>auto</cpu_mitigations>
+      <gfxmode>auto</gfxmode>
+      <hiddenmenu>false</hiddenmenu>
+      <os_prober>false</os_prober>
+      <serial>serial --unit=1 --speed=9600 --parity=no</serial>
```

Comment:
Guess 9600 baud is the default for s390x instead of 115200 used elsewhere?

Reply:
I have noticed. Same reason as above. I did run some test with differences in the profile but not for all the cases.